lib/y2_base.pm

```diff
@@ -160,6 +160,9 @@ sub upload_widgets_json {
 
 sub post_fail_hook {
     my $self = shift;
+
+    return if (get_var('NOLOGS'));
+
```

Comment:
I consider this too verbose. Just leave out the blank lines please. This applies for all changes